rendered/zip-0312.html

```diff
@@ -10,11 +10,11 @@
         <pre>ZIP: 312
 Title: FROST for Spend Authorization Multisignatures
 Owners: Conrado Gouvea &lt;conrado@zfnd.org&gt;
-        Chelsea Komlo &lt;ckomlo@uwaterloo.ca&gt;
-        Deirdre Connolly &lt;deirdre@zfnd.org&gt;
+Original-Authors: Chelsea Komlo
+                  Deirdre Connolly
 Status: Draft
 Category: Wallet
-Created: 2022-08-dd
+Created: 2024-08-dd
```

Comment:
This can just be the date it was originally committed which was apparently 2023-01-12. https://github.com/zcash/zips/commit/48f8c2c80e890385a9619e4dbf3b62ff8963b010
```suggestion
Created: 2023-01-12
```